rest/taskana-rest-spring/src/main/java/pro/taskana/common/rest/RestEndpoints.java

```diff
@@ -14,7 +14,7 @@
   public static final String URL_CLASSIFICATION_CATEGORIES_BY_TYPES =
       API_V1 + "classifications-by-type";
   public static final String URL_HISTORY_ENABLED = API_V1 + "history-provider-enabled";
-  public static final String URL_CUSTOM_ATTRIBUTES = API_V1 + "/config/custom-attributes";
+  public static final String URL_CUSTOM_ATTRIBUTES = API_V1 + "config/custom-attributes";
```

Comment:
Isn't this a breaking change? Atleast our FrontEnd uses this endpoint to also update custom-attributes I'd assume?

Reply:
This slash was redundant as "API_V1" already ends with a slash. The new Spring Boot version obviously is more strict and does not like double slashes...